src/components/Card/Card.module.scss

```
@@ -0,0 +1,58 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  padding: 8px 12px;
+  width: 155px;
+  position: relative;
+  background: #FFFFFF;
+  box-shadow: 0px 1px 8px rgba(0, 0, 0, 0.12);
+  border-radius: 10px;
+}
+
+.card:hover {
+  opacity: 0.8;
+  background: #f3f3f3;
+}
+
+.card__image {
+  margin: 3px 22px 3px 22px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 25px;
+  color: #0D0D0D;
+}
+
+.card__subtitle {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 16px;
+  color: #3B3B3B;
+  margin-bottom: 7px;
+}
+
+.card__caloriesMore {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: auto 0 0 0;
+}
+
+.card__calories {
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 18px;
+  color: #FF0000;
+}
+
+.card__more {
+  width: 24px;
+  height: 24px;
+  background-image: url(../../../public/more.svg);

```

здесь и где ещё встретишь:
```suggestion
  background-image: url(~/public/more.svg);
```

---

Спасибо! Выглядело это страшно, согласен, гуглил как можно сделать красиво, но что-то не находил